docs/src/components/RocketImage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { useStaticQuery, graphql } from "gatsby";
+import Img, { FluidObject } from "gatsby-image";
+import { css } from "styled-components";
+
+export default function RocketImage() {
+  const data: {
+    file: {
+      childImageSharp: {
+        fluid: FluidObject;
+      };
+    };
+  } = useStaticQuery(graphql`
+    query {
+      file(relativePath: { eq: "rocket.jpg" }) {
+        childImageSharp {
```

Comment:
can we add also something like: 
```
srcWebp
srcSetWebp
tracedSVG
```
?)

Reply:
I'm not a big fan of traced SVG, especially for this purpose, I prefer current blurred placeholder. But yes to WebP! Any image optimizations are welcome, as long as the loading experience is good. 👍